src/cmdlib.sh

```diff
@@ -387,6 +387,13 @@ EOF
         done
     fi
 
+    local imagejsondir="${tmp_overridesdir}/imagejson"
+    export ostree_image_json="/usr/share/coreos-assembler/image.json"
```

Comment:
Related to Dusty's comment above, it doesn't seem like this is used anywhere else, so let's just make it a `local` too?